velox/expression/tests/CastExprTest.cpp

```diff
@@ -348,6 +349,26 @@ TEST_F(CastExprTest, timestampToDate) {
       false,
       TIMESTAMP(),
       DATE());
+
+  setTimezone("America/Los_Angeles");
+  testCast<Timestamp, int32_t>(
+      "date",
+      {
+          Timestamp(0, 0),
```

Comment:
The input list is the same in both the tests (which is good) so abstract it as a common variable.